src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import Badge from '~/components/Badge.astro';
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build your Astro site with any code editor, installed locally or in the cloud!
+
+This page explains how to improve the experience of editing `.astro` files in supported editors.
+
+## VS Code Extension
+
+[VS Code](https://code.visualstudio.com) is a popular code editor that can be downloaded locally, and is also used by online platforms such as [GitHub Codespaces](https://github.com/features/codespaces), and [Gitpod](https://gitpod.io). 
+
+Astro has its own [Astro VS Code Extension on Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) that you can install in any of these VS Code environments for .`astro` syntax highligting, code completion, and more!
+
+### Full List of Features
+- [Go to Definition](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition)
```

Comment:
Either all these get links or they dont, 
```suggestion
- [Go to Definition](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition)
```

Reply:
I did this because in our official extension page, it has a link (and only this one). I've also seen it (and only it) linked in a feature set elsewhere. So, I don't know if this is a new thing, or it's special or what. But, that's what we do in the VS Marketplace/Open VSX registry, so there's precedent.

Reply:
To be honest it's a bit of a legacy list from the time where the features supported were really precise. Nowadays, we have *almost* (I'm working on it!) all the "basic" features like these and the only features we would really need to list are those that are Astro specific